barretenberg/cpp/pil/avm/main.pil

```diff
@@ -16,6 +16,17 @@ include "gadgets/mem_slice.pil";
 include "gadgets/merkle_tree.pil";
 
 namespace main(256);
+    //===== PUBLIC INPUT POLYNOMIALS ======================================
```

Comment:
These are moved over to just keep the existing public inputs structure and is not representative of the final version

Reply:
yeah - the old pil file (kernel.pil) is commented out - it will eventually be deleted entirely, but ive kept it because we will need to reference / copy stuff from it when we constrain the new inputs